<!DOCTYPE html>
<html lang="zh_CN">
<head>
    <meta charset="zh_CN">
    <title>肝癌cfRNA检测报告</title>
    <link rel="stylesheet" type="text/css" href="css/body.css">
    <script type="text/javascript" async
        src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js">
    </script>
</head>

<body>   
    <div class="centered-div">
        <div class="logo">
            <!-- <a href=src/img/logo.png> -->
            <!-- <img src=src/img/logo.png alt="公司logo"  height="100px" width="180px"> -->
            <img src=src/img/logo.png alt="公司logo">
            <!-- </a> -->
        </div>

        <h1>肝癌cfRNA检测报告</h1>
        <!-- <br> -->
        <hr class="solid-line" align="center">

        <br>
        <h2>背景介绍</h2>
        <p>随着高通量测序技术的不断进步，循环游离RNA（Cell-free RNA）已在疾病发生的机制研究、生物标志物的筛选以及药物研发等领域发挥出巨大的临床应用潜力。已经有很多研究证明了循环游离RNA在妊娠、肿瘤、神经退行性疾病等领域的预测、早期筛查、辅助诊断、治疗效果监测等方面的应用潜力。cfRNA 由多种类型的 RNA 构成，根据分子长度的不同，cfRNA大致可以划分为两大类：小RNA和长RNA。小RNA长度约为 18-40 nt，最长不超过 200 nt。因为它们都是非编码 RNA，因此也被称为 sncRNA（Small Non-coding RNA）。cfRNA 当中的小 RNA 包括以下几种主要亚型：微小RNA（miRNA）、piwi相互作用RNA（piRNA）、tRNA衍生的小RNA（tsRNA）还有 rRNA 衍生的小 RNA（rsRNA）。另一方面，长RNA的长度则往往可以达到数百到数千nt，主要包括mRNA、长非编码RNA（lncRNA）和环状RNA（circRNA）。</p>
        <hr class="solid-line" align="center">

        <br>
        <h2>一、实验流程</h2>
        <p></p>
        <hr class="solid-line" align="center">

        <br>
        <h2>二、生物信息学分析</h2>
        <br>
        <h3>2.1 生物信息流程</h3>
        <p>将下机数据进行过滤得到Clean Data，与各种RNA类型的数据库比对，分离不同的RNA类型，根据不同的RNA类型做差异分析。将差异分析的结果通过机器学习的方法得到候选基因，同时收集文献已经发表的gene进行筛选片段评估。一种方式是k-mer统计，是以bin的方式扫描样本在候选基因上的覆盖深度，筛选符合要求的区间；另一种是call peak方式，统计样本在基因上的峰值区域，找出reads在基因上覆盖的顶峰值，向两边延伸到峰值的一半截至的区间。对候选的区域设置多种评估指标作为过滤标准，经过进一步筛选后，进行qPCR引物设计。</p>
        <p>cfRNA分析流程如下：</p>
        <div class="Fixed-image">
            <a href=src/img/pipeline.png target="_blank">
                <img src=src/img/pipeline.png width="540px" height="700px">
            </a>
        </div>
    









<!--============================== 质控 ================================ -->
        <br>
        <h3>2.2 质控</h3>
        <h4>2.2.1 测序数据及其质量控制</h4>
        <p>基于边合成边测序（Sequencing By Synthesis，SBS）技术，Illumina 高通量测序平台对cDNA文库进行测序，产出大量的高质量Data，称为原始数据（Raw Data），其大部分碱基质量打分能达到或超过Q30。</p>
        <p>Raw Data通常以FASTQ格式提供，每个测序样品的Raw Data包括两个FASTQ文件，分别包含所有cDNA片段两端测定的Reads。</p>
        <p>FASTQ格式文件示意图如下：</p>
        <div class="Fixed-image">
            <a href=src/img/质控.png target="_blank">
                <img src=src/img/质控.png width="800px" height="300px">
            </a>
        </div>
        <p class="p_img">注：FASTQ文件中通常每4行对应一个序列单元：</p>
        <p class="p_img">（1）第一行以@开头，后面接着序列标识（ID）以及其它可选的描述信息；</p>
        <p class="p_img">（2）第二行为碱基序列，即Reads；</p>
        <p class="p_img">（3）第三行以“+”开头，后面接着可选的描述信息；</p>
        <p class="p_img">（4）第四行为Reads每个碱基对应的质量打分编码，长度必须和Reads的序列长度相同，该行中每个字符对应的</p>
        


        <h4>2.2.2 测序碱基质量分布</h4>
        <p>碱基质量分布检查用于检测整体样本的reads碱基有没有出现某些区段较低的情况，以此来排除异常</p>
        <div class="image-container">
            <div class="side_by_side_img">
                <a href=src/img/1.QC/base_R1.png target="_blank">
                    <img src=src/img/1.QC/base_R1.png alt="Image 1">
                    <p>R1</p>
                </a>    
            </div>
            <div class="side_by_side_img">
                <a href=src/img/1.QC/base_R2.png target="_blank">
                    <img src=src/img/1.QC/base_R2.png alt="Image 2">
                    <p>R2</p>
                </a>
            </div>
        </div>
        <p style="text-align: center;">图 碱基错误率分布图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>


        <h4>2.2.3 测序碱基含量分布</h4>
        <p>碱基类型分布检查用于检测有无AT、GC分离现象,由于RNA-Seq所测的序列为随机打断的cDNA片段，因随机性打断及碱基互补配对原则，理论上，G和C、A和T的含量每个测序循环上应分别相等，且整个测序过程稳定不变，呈水平线。由于Reads 5’端的前几个碱基为随机引物序列存在一定的偏好性，因此会在碱基分布图中出现前端波动较大的现象。</p>
        <div class="image-container">
            <div class="side_by_side_img">
                <a href=src/img/1.QC/GC_R1.png target="_blank">
                    <img src=src/img/1.QC/GC_R1.png alt="Image 1">
                    <p>R1</p>
                </a>    
            </div>
            <div class="side_by_side_img">
                <a href=src/img/1.QC/GC_R2.png target="_blank">
                    <img src=src/img/1.QC/GC_R2.png alt="Image 2">
                    <p>R2</p>
                </a>
            </div>
        </div>
        <p style="text-align: center;">图 ATCG含量分布图</p>
        <p class="p_img">注：横坐标为Reads的碱基位置，纵坐标为单碱基所占比例。</p>



        <h4>2.2.4 测序数据统计方法</h4>
        <p>在进行数据分析之前，首先需要确保这些Reads有足够高的质量，以保证后续分析的准确。百迈客对数据进行严格的质量控制，进行如下过滤方式：</p>
        <p>(1) 去除含有接头的Reads；</p>
        <p>(2) 去除低质量的Reads（去除质量值Q&lt;=25的碱基数占整条Read的50%以上的Reads，同时去除长度低于15bp的Reads）</p>

        

        <h4>2.2.5 测序数据统计结果</h4>
        <p>该项目各样品数据产出统计见下表：！！！这里要改一下，添加Q20,Q30</p>
        <div class="scrollable-div">
            <table>
                <tr>
                    <th>Sample</th>
                    <th>Raw_Seq_Num</th>
                    <th>Raw_bases_Num(Gbp)</th>
                    <th>Clean_Seq_Num</th>
                    <th>Clean_bases_Num(Gbp)</th>
                    <th>Q20</th>
                    <th>Q30</th>
                </tr>
                
                <tr>
                    <td>SRR10822575</td>
                    <td>65337632</td>
                    <td>8.2</td>
                    <td>65334782 (100.0%)</td>
                    <td>8.0</td>
                </tr>
                
                <tr>
                    <td>SRR10822576</td>
                    <td>75706840</td>
                    <td>10.58</td>
                    <td>75701066 (99.99%)</td>
                    <td>10.4</td>
                </tr>
                
                <tr>
                    <td>SRR10822577</td>
                    <td>93456838</td>
                    <td>13.17</td>
                    <td>93449402 (99.99%)</td>
                    <td>13.0</td>
                </tr>
                
                <tr>
                    <td>SRR10822578</td>
                    <td>56007186</td>
                    <td>7.54</td>
                    <td>56001642 (99.99%)</td>
                    <td>7.4</td>
                </tr>
                
                <tr>
                    <td>SRR10822579</td>
                    <td>64391146</td>
                    <td>8.55</td>
                    <td>64385632 (99.99%)</td>
                    <td>8.4</td>
                </tr>
                
                <tr>
                    <td>SRR14506843</td>
                    <td>70390124</td>
                    <td>10.56</td>
                    <td>69512138 (98.75%)</td>
                    <td>9.0</td>
                </tr>
                
                <tr>
                    <td>SRR14506844</td>
                    <td>66489580</td>
                    <td>9.97</td>
                    <td>65608700 (98.68%)</td>
                    <td>9.0</td>
                </tr>
                
                <tr>
                    <td>SRR14506845</td>
                    <td>75501220</td>
                    <td>11.33</td>
                    <td>72355048 (95.83%)</td>
                    <td>9.7</td>
                </tr>
                
                <tr>
                    <td>SRR14506846</td>
                    <td>70032328</td>
                    <td>10.5</td>
                    <td>68120686 (97.27%)</td>
                    <td>6.0</td>
                </tr>
                
                <tr>
                    <td>SRR14506847</td>
                    <td>60265886</td>
                    <td>9.04</td>
                    <td>56450680 (93.67%)</td>
                    <td>3.4</td>
                </tr>
                
                <tr>
                    <td>SRR10822540</td>
                    <td>52993626</td>
                    <td>7.1</td>
                    <td>52991296 (100.0%)</td>
                    <td>7.0</td>
                </tr>
                
                <tr>
                    <td>SRR10822541</td>
                    <td>54714074</td>
                    <td>7.39</td>
                    <td>54711834 (100.0%)</td>
                    <td>7.2</td>
                </tr>
                
                <tr>
                    <td>SRR10822542</td>
                    <td>29119042</td>
                    <td>3.87</td>
                    <td>29117896 (100.0%)</td>
                    <td>3.8</td>
                </tr>
                
                <tr>
                    <td>SRR10822543</td>
                    <td>71936216</td>
                    <td>9.77</td>
                    <td>71933378 (100.0%)</td>
                    <td>9.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822544</td>
                    <td>53701338</td>
                    <td>7.13</td>
                    <td>53698428 (99.99%)</td>
                    <td>7.0</td>
                </tr>
                
                <tr>
                    <td>SRR10822545</td>
                    <td>68843344</td>
                    <td>8.86</td>
                    <td>68841568 (100.0%)</td>
                    <td>8.7</td>
                </tr>
                
                <tr>
                    <td>SRR10822546</td>
                    <td>63023340</td>
                    <td>8.71</td>
                    <td>63019198 (99.99%)</td>
                    <td>8.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822547</td>
                    <td>55550830</td>
                    <td>7.46</td>
                    <td>55547418 (99.99%)</td>
                    <td>7.4</td>
                </tr>
                
                <tr>
                    <td>SRR10822548</td>
                    <td>49912116</td>
                    <td>7.05</td>
                    <td>49908924 (99.99%)</td>
                    <td>6.9</td>
                </tr>
                
                <tr>
                    <td>SRR10822549</td>
                    <td>63142008</td>
                    <td>8.74</td>
                    <td>63135776 (99.99%)</td>
                    <td>8.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822550</td>
                    <td>57881544</td>
                    <td>7.71</td>
                    <td>57878364 (99.99%)</td>
                    <td>7.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822551</td>
                    <td>83075152</td>
                    <td>10.74</td>
                    <td>83071010 (100.0%)</td>
                    <td>10.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822552</td>
                    <td>63348396</td>
                    <td>8.92</td>
                    <td>63346354 (100.0%)</td>
                    <td>8.7</td>
                </tr>
                
                <tr>
                    <td>SRR10822553</td>
                    <td>54822520</td>
                    <td>7.36</td>
                    <td>54819252 (99.99%)</td>
                    <td>7.2</td>
                </tr>
                
                <tr>
                    <td>SRR10822554</td>
                    <td>59414074</td>
                    <td>8.0</td>
                    <td>59411238 (100.0%)</td>
                    <td>7.8</td>
                </tr>
                
                <tr>
                    <td>SRR10822555</td>
                    <td>75774574</td>
                    <td>9.8</td>
                    <td>75771798 (100.0%)</td>
                    <td>9.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822556</td>
                    <td>66877916</td>
                    <td>9.37</td>
                    <td>66873876 (99.99%)</td>
                    <td>9.2</td>
                </tr>
                
                <tr>
                    <td>SRR10822557</td>
                    <td>64700674</td>
                    <td>8.95</td>
                    <td>64695906 (99.99%)</td>
                    <td>8.8</td>
                </tr>
                
                <tr>
                    <td>SRR10822558</td>
                    <td>61469804</td>
                    <td>8.47</td>
                    <td>61465994 (99.99%)</td>
                    <td>8.4</td>
                </tr>
                
                <tr>
                    <td>SRR10822559</td>
                    <td>66144156</td>
                    <td>9.14</td>
                    <td>66141276 (100.0%)</td>
                    <td>9.0</td>
                </tr>
                
                <tr>
                    <td>SRR10822560</td>
                    <td>77127692</td>
                    <td>10.73</td>
                    <td>77123706 (99.99%)</td>
                    <td>10.5</td>
                </tr>
                
                <tr>
                    <td>SRR10822561</td>
                    <td>74212610</td>
                    <td>9.64</td>
                    <td>74209200 (100.0%)</td>
                    <td>9.5</td>
                </tr>
                
                <tr>
                    <td>SRR10822562</td>
                    <td>76207052</td>
                    <td>9.67</td>
                    <td>76203608 (100.0%)</td>
                    <td>9.5</td>
                </tr>
                
                <tr>
                    <td>SRR10822563</td>
                    <td>63899000</td>
                    <td>8.77</td>
                    <td>63896072 (100.0%)</td>
                    <td>8.7</td>
                </tr>
                
                <tr>
                    <td>SRR10822564</td>
                    <td>66827972</td>
                    <td>9.24</td>
                    <td>66823890 (99.99%)</td>
                    <td>9.1</td>
                </tr>
                
                <tr>
                    <td>SRR10822566</td>
                    <td>78943032</td>
                    <td>10.85</td>
                    <td>78938902 (99.99%)</td>
                    <td>10.7</td>
                </tr>
                
                <tr>
                    <td>SRR10822567</td>
                    <td>76449964</td>
                    <td>10.26</td>
                    <td>76446572 (100.0%)</td>
                    <td>10.1</td>
                </tr>
                
                <tr>
                    <td>SRR10822568</td>
                    <td>69919914</td>
                    <td>9.69</td>
                    <td>69916188 (99.99%)</td>
                    <td>9.5</td>
                </tr>
                
                <tr>
                    <td>SRR10822569</td>
                    <td>73489862</td>
                    <td>9.7</td>
                    <td>73486732 (100.0%)</td>
                    <td>9.6</td>
                </tr>
                
                <tr>
                    <td>SRR10822570</td>
                    <td>62910640</td>
                    <td>8.27</td>
                    <td>62908534 (100.0%)</td>
                    <td>8.1</td>
                </tr>
                
            </table>
        </div>
        <p class="p_img">注：（1）Sample：样本的名称</p>
        <p class="p_img">（2）Raw_Seq_Num：样本的raw数据的reads数目</p>
        <p class="p_img">（3）Raw_bases_Num(Gbp)：样本的raw数据的bases数目</p>
        <p class="p_img">（4）Clean_Seq_Num：样本的clean数据的reads数目</p>
        <p class="p_img">（5）Clean_bases_Num(Gbp)：样本的clean数据的bases数目</p>
        









<!--============================== 序列比对 ================================ -->
        <br>
        <h3>2.3 cfRNA数据与参考基因组序列比对</h3>
        <p>本项目使用的人类基因组版本是GRCh38.p13，gff注释版本为Gencode.v34。</p>
        <p>使用STAR对mRNA和lncRNA进行比对，这是由于STAR拥有极高的比对速度，同时也具有很高的准确度，STAR算法包括两个主要的步骤：seed searching step 和 clustering/stitching/scoring step。这些算法令其成为RNA高通量序列比对的最好选择之一。</p>
        <p>使用bowtie2对miRNA进行比对</p>
        <p>统计结果在：。。。。，比对结果见下表：</p>
        <div class="scrollable-div">
            <table>
                <tr>
                    <th>Sample</th>
                    <th>Clean_Seq_Num</th>
                    <th>Clean_bases_Num(Gbp)</th>
                    <th>UniVec_Seq_Num</th>
                    <th>rRNA_Seq_Num</th>
                    <th>miRNA_Seq_Num</th>
                    <th>tRNA_Seq_Num</th>
                    <th>piRNA_Seq_Num</th>
                    <th>Genome_Seq_Num</th>
                    <th>Genome_Rmdup_Seq_Num</th>
                </tr>
                
                <tr>
                    <td>SRR10822575</td>
                    <td>65334782 (100.0%)</td>
                    <td>8.0</td>
                    <td>1804852 (2.76%)</td>
                    <td>59640 (0.09%)</td>
                    <td>742 (0.0%)</td>
                    <td>340 (0.0%)</td>
                    <td>8096 (0.01%)</td>
                    <td>17147889 (27.02%)</td>
                    <td>4863676 (28.36%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822576</td>
                    <td>75701066 (99.99%)</td>
                    <td>10.4</td>
                    <td>390540 (0.52%)</td>
                    <td>115400 (0.15%)</td>
                    <td>62 (0.0%)</td>
                    <td>20 (0.0%)</td>
                    <td>512 (0.0%)</td>
                    <td>24856008 (33.06%)</td>
                    <td>3361462 (13.52%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822577</td>
                    <td>93449402 (99.99%)</td>
                    <td>13.0</td>
                    <td>357854 (0.38%)</td>
                    <td>82868 (0.09%)</td>
                    <td>66 (0.0%)</td>
                    <td>4 (0.0%)</td>
                    <td>646 (0.0%)</td>
                    <td>18342300 (19.72%)</td>
                    <td>3141133 (17.13%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822578</td>
                    <td>56001642 (99.99%)</td>
                    <td>7.4</td>
                    <td>477984 (0.85%)</td>
                    <td>87746 (0.16%)</td>
                    <td>50 (0.0%)</td>
                    <td>22 (0.0%)</td>
                    <td>1440 (0.0%)</td>
                    <td>15668474 (28.26%)</td>
                    <td>3484277 (22.24%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822579</td>
                    <td>64385632 (99.99%)</td>
                    <td>8.4</td>
                    <td>284216 (0.44%)</td>
                    <td>69342 (0.11%)</td>
                    <td>160 (0.0%)</td>
                    <td>14 (0.0%)</td>
                    <td>2144 (0.0%)</td>
                    <td>24068999 (37.59%)</td>
                    <td>7195503 (29.9%)</td>
                </tr>
                
                <tr>
                    <td>SRR14506843</td>
                    <td>69512138 (98.75%)</td>
                    <td>9.0</td>
                    <td>889508 (1.28%)</td>
                    <td>87038 (0.13%)</td>
                    <td>2736 (0.0%)</td>
                    <td>198 (0.0%)</td>
                    <td>66458 (0.1%)</td>
                    <td>18120420 (26.47%)</td>
                    <td>3125199 (17.25%)</td>
                </tr>
                
                <tr>
                    <td>SRR14506844</td>
                    <td>65608700 (98.68%)</td>
                    <td>9.0</td>
                    <td>528304 (0.81%)</td>
                    <td>121542 (0.19%)</td>
                    <td>468 (0.0%)</td>
                    <td>78 (0.0%)</td>
                    <td>11800 (0.02%)</td>
                    <td>20542655 (31.63%)</td>
                    <td>3372372 (16.42%)</td>
                </tr>
                
                <tr>
                    <td>SRR14506845</td>
                    <td>72355048 (95.83%)</td>
                    <td>9.7</td>
                    <td>444714 (0.61%)</td>
                    <td>25904 (0.04%)</td>
                    <td>1224 (0.0%)</td>
                    <td>336 (0.0%)</td>
                    <td>29702 (0.04%)</td>
                    <td>10416655 (14.5%)</td>
                    <td>692802 (6.65%)</td>
                </tr>
                
                <tr>
                    <td>SRR14506846</td>
                    <td>68120686 (97.27%)</td>
                    <td>6.0</td>
                    <td>2786960 (4.09%)</td>
                    <td>13653512 (20.9%)</td>
                    <td>20152 (0.04%)</td>
                    <td>40030 (0.08%)</td>
                    <td>645238 (1.25%)</td>
                    <td>5039948 (9.89%)</td>
                    <td>639396 (12.69%)</td>
                </tr>
                
                <tr>
                    <td>SRR14506847</td>
                    <td>56450680 (93.67%)</td>
                    <td>3.4</td>
                    <td>1109440 (1.97%)</td>
                    <td>11874606 (21.46%)</td>
                    <td>51438 (0.12%)</td>
                    <td>43468 (0.1%)</td>
                    <td>1643736 (3.79%)</td>
                    <td>6690442 (16.03%)</td>
                    <td>783364 (11.71%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822540</td>
                    <td>52991296 (100.0%)</td>
                    <td>7.0</td>
                    <td>178246 (0.34%)</td>
                    <td>44452 (0.08%)</td>
                    <td>14 (0.0%)</td>
                    <td>6 (0.0%)</td>
                    <td>478 (0.0%)</td>
                    <td>32282822 (61.18%)</td>
                    <td>3613150 (11.19%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822541</td>
                    <td>54711834 (100.0%)</td>
                    <td>7.2</td>
                    <td>146724 (0.27%)</td>
                    <td>110810 (0.2%)</td>
                    <td>80 (0.0%)</td>
                    <td>12 (0.0%)</td>
                    <td>458 (0.0%)</td>
                    <td>30682099 (56.35%)</td>
                    <td>2702734 (8.81%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822542</td>
                    <td>29117896 (100.0%)</td>
                    <td>3.8</td>
                    <td>135234 (0.46%)</td>
                    <td>20638 (0.07%)</td>
                    <td>110 (0.0%)</td>
                    <td>4 (0.0%)</td>
                    <td>348 (0.0%)</td>
                    <td>12724446 (43.94%)</td>
                    <td>726364 (5.71%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822543</td>
                    <td>71933378 (100.0%)</td>
                    <td>9.6</td>
                    <td>39852 (0.06%)</td>
                    <td>39920 (0.06%)</td>
                    <td>136 (0.0%)</td>
                    <td>208 (0.0%)</td>
                    <td>4446 (0.01%)</td>
                    <td>50567859 (70.38%)</td>
                    <td>21318421 (42.16%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822544</td>
                    <td>53698428 (99.99%)</td>
                    <td>7.0</td>
                    <td>54894 (0.1%)</td>
                    <td>12752 (0.02%)</td>
                    <td>16 (0.0%)</td>
                    <td>0 (0.0%)</td>
                    <td>890 (0.0%)</td>
                    <td>34762361 (64.82%)</td>
                    <td>4449247 (12.8%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822545</td>
                    <td>68841568 (100.0%)</td>
                    <td>8.7</td>
                    <td>1765680 (2.56%)</td>
                    <td>31520 (0.05%)</td>
                    <td>392 (0.0%)</td>
                    <td>30 (0.0%)</td>
                    <td>966 (0.0%)</td>
                    <td>14684783 (21.9%)</td>
                    <td>818918 (5.58%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822546</td>
                    <td>63019198 (99.99%)</td>
                    <td>8.6</td>
                    <td>162022 (0.26%)</td>
                    <td>47844 (0.08%)</td>
                    <td>80 (0.0%)</td>
                    <td>40 (0.0%)</td>
                    <td>1082 (0.0%)</td>
                    <td>40644437 (64.71%)</td>
                    <td>12479979 (30.71%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822547</td>
                    <td>55547418 (99.99%)</td>
                    <td>7.4</td>
                    <td>247344 (0.45%)</td>
                    <td>58582 (0.11%)</td>
                    <td>334 (0.0%)</td>
                    <td>82 (0.0%)</td>
                    <td>1714 (0.0%)</td>
                    <td>30367084 (54.97%)</td>
                    <td>14801983 (48.74%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822548</td>
                    <td>49908924 (99.99%)</td>
                    <td>6.9</td>
                    <td>11402 (0.02%)</td>
                    <td>37984 (0.08%)</td>
                    <td>34 (0.0%)</td>
                    <td>20 (0.0%)</td>
                    <td>578 (0.0%)</td>
                    <td>34836488 (69.87%)</td>
                    <td>5881678 (16.88%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822549</td>
                    <td>63135776 (99.99%)</td>
                    <td>8.6</td>
                    <td>143138 (0.23%)</td>
                    <td>73516 (0.12%)</td>
                    <td>80 (0.0%)</td>
                    <td>162 (0.0%)</td>
                    <td>1310 (0.0%)</td>
                    <td>40731225 (64.74%)</td>
                    <td>9970419 (24.48%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822550</td>
                    <td>57878364 (99.99%)</td>
                    <td>7.6</td>
                    <td>70200 (0.12%)</td>
                    <td>38022 (0.07%)</td>
                    <td>114 (0.0%)</td>
                    <td>24 (0.0%)</td>
                    <td>1626 (0.0%)</td>
                    <td>36456138 (63.11%)</td>
                    <td>3768156 (10.34%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822551</td>
                    <td>83071010 (100.0%)</td>
                    <td>10.6</td>
                    <td>294412 (0.35%)</td>
                    <td>56750 (0.07%)</td>
                    <td>982 (0.0%)</td>
                    <td>124 (0.0%)</td>
                    <td>4294 (0.01%)</td>
                    <td>36183296 (43.74%)</td>
                    <td>19285331 (53.3%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822552</td>
                    <td>63346354 (100.0%)</td>
                    <td>8.7</td>
                    <td>18564 (0.03%)</td>
                    <td>28828 (0.05%)</td>
                    <td>70 (0.0%)</td>
                    <td>36 (0.0%)</td>
                    <td>1036 (0.0%)</td>
                    <td>42401345 (66.99%)</td>
                    <td>13488395 (31.81%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822553</td>
                    <td>54819252 (99.99%)</td>
                    <td>7.2</td>
                    <td>169648 (0.31%)</td>
                    <td>50116 (0.09%)</td>
                    <td>76 (0.0%)</td>
                    <td>2 (0.0%)</td>
                    <td>580 (0.0%)</td>
                    <td>30268139 (55.44%)</td>
                    <td>2411757 (7.97%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822554</td>
                    <td>59411238 (100.0%)</td>
                    <td>7.8</td>
                    <td>21402 (0.04%)</td>
                    <td>60572 (0.1%)</td>
                    <td>56 (0.0%)</td>
                    <td>146 (0.0%)</td>
                    <td>2460 (0.0%)</td>
                    <td>40352721 (68.02%)</td>
                    <td>14703828 (36.44%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822555</td>
                    <td>75771798 (100.0%)</td>
                    <td>9.6</td>
                    <td>302862 (0.4%)</td>
                    <td>53380 (0.07%)</td>
                    <td>928 (0.0%)</td>
                    <td>100 (0.0%)</td>
                    <td>4050 (0.01%)</td>
                    <td>37252030 (49.4%)</td>
                    <td>18373615 (49.32%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822556</td>
                    <td>66873876 (99.99%)</td>
                    <td>9.2</td>
                    <td>127648 (0.19%)</td>
                    <td>10604 (0.02%)</td>
                    <td>50 (0.0%)</td>
                    <td>8 (0.0%)</td>
                    <td>728 (0.0%)</td>
                    <td>46229754 (69.27%)</td>
                    <td>20611156 (44.58%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822557</td>
                    <td>64695906 (99.99%)</td>
                    <td>8.8</td>
                    <td>98666 (0.15%)</td>
                    <td>54964 (0.09%)</td>
                    <td>90 (0.0%)</td>
                    <td>116 (0.0%)</td>
                    <td>1242 (0.0%)</td>
                    <td>43046273 (66.7%)</td>
                    <td>11173043 (25.96%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822558</td>
                    <td>61465994 (99.99%)</td>
                    <td>8.4</td>
                    <td>62802 (0.1%)</td>
                    <td>31874 (0.05%)</td>
                    <td>98 (0.0%)</td>
                    <td>220 (0.0%)</td>
                    <td>3058 (0.0%)</td>
                    <td>42132975 (68.66%)</td>
                    <td>7832096 (18.59%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822559</td>
                    <td>66141276 (100.0%)</td>
                    <td>9.0</td>
                    <td>76644 (0.12%)</td>
                    <td>27388 (0.04%)</td>
                    <td>80 (0.0%)</td>
                    <td>56 (0.0%)</td>
                    <td>1876 (0.0%)</td>
                    <td>45811755 (69.37%)</td>
                    <td>16212713 (35.39%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822560</td>
                    <td>77123706 (99.99%)</td>
                    <td>10.5</td>
                    <td>39036 (0.05%)</td>
                    <td>38758 (0.05%)</td>
                    <td>78 (0.0%)</td>
                    <td>154 (0.0%)</td>
                    <td>2170 (0.0%)</td>
                    <td>53740405 (69.75%)</td>
                    <td>12611519 (23.47%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822561</td>
                    <td>74209200 (100.0%)</td>
                    <td>9.5</td>
                    <td>180042 (0.24%)</td>
                    <td>83822 (0.11%)</td>
                    <td>114 (0.0%)</td>
                    <td>292 (0.0%)</td>
                    <td>3402 (0.0%)</td>
                    <td>40953648 (55.39%)</td>
                    <td>13371491 (32.65%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822562</td>
                    <td>76203608 (100.0%)</td>
                    <td>9.5</td>
                    <td>138792 (0.18%)</td>
                    <td>17864 (0.02%)</td>
                    <td>90 (0.0%)</td>
                    <td>22 (0.0%)</td>
                    <td>2216 (0.0%)</td>
                    <td>22963872 (30.2%)</td>
                    <td>9988539 (43.5%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822563</td>
                    <td>63896072 (100.0%)</td>
                    <td>8.7</td>
                    <td>47752 (0.07%)</td>
                    <td>125886 (0.2%)</td>
                    <td>198 (0.0%)</td>
                    <td>30 (0.0%)</td>
                    <td>1378 (0.0%)</td>
                    <td>41778527 (65.56%)</td>
                    <td>9281091 (22.21%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822564</td>
                    <td>66823890 (99.99%)</td>
                    <td>9.1</td>
                    <td>108562 (0.16%)</td>
                    <td>51382 (0.08%)</td>
                    <td>100 (0.0%)</td>
                    <td>34 (0.0%)</td>
                    <td>1128 (0.0%)</td>
                    <td>44243141 (66.37%)</td>
                    <td>11847596 (26.78%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822566</td>
                    <td>78938902 (99.99%)</td>
                    <td>10.7</td>
                    <td>83428 (0.11%)</td>
                    <td>102240 (0.13%)</td>
                    <td>198 (0.0%)</td>
                    <td>120 (0.0%)</td>
                    <td>1442 (0.0%)</td>
                    <td>50901251 (64.64%)</td>
                    <td>11320965 (22.24%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822567</td>
                    <td>76446572 (100.0%)</td>
                    <td>10.1</td>
                    <td>278284 (0.36%)</td>
                    <td>82700 (0.11%)</td>
                    <td>418 (0.0%)</td>
                    <td>68 (0.0%)</td>
                    <td>2602 (0.0%)</td>
                    <td>43706745 (57.45%)</td>
                    <td>17366739 (39.73%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822568</td>
                    <td>69916188 (99.99%)</td>
                    <td>9.5</td>
                    <td>121366 (0.17%)</td>
                    <td>83406 (0.12%)</td>
                    <td>114 (0.0%)</td>
                    <td>136 (0.0%)</td>
                    <td>1562 (0.0%)</td>
                    <td>46539674 (66.76%)</td>
                    <td>9411310 (20.22%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822569</td>
                    <td>73486732 (100.0%)</td>
                    <td>9.6</td>
                    <td>43272 (0.06%)</td>
                    <td>69932 (0.1%)</td>
                    <td>868 (0.0%)</td>
                    <td>284 (0.0%)</td>
                    <td>2518 (0.0%)</td>
                    <td>41863567 (57.06%)</td>
                    <td>13664170 (32.64%)</td>
                </tr>
                
                <tr>
                    <td>SRR10822570</td>
                    <td>62908534 (100.0%)</td>
                    <td>8.1</td>
                    <td>60044 (0.1%)</td>
                    <td>25490 (0.04%)</td>
                    <td>62 (0.0%)</td>
                    <td>70 (0.0%)</td>
                    <td>2618 (0.0%)</td>
                    <td>39503969 (62.88%)</td>
                    <td>6811377 (17.24%)</td>
                </tr>
                
            </table>
        </div>
        <p class="p_img">注：（1）Sample：样本的名称</p>
        <p class="p_img">（2）Clean_Seq_Num：样本的clean数据的reads数目</p>
        <p class="p_img">（3）Clean_bases_Num(Gbp)：样本的clean数据的bases数目</p>
        <p class="p_img">（4）*_Seq_Num：各种RNA类型数据的reads数目以及其占clean reads数目的比例</p>
        <p class="p_img">（5）Genome_Seq_Num：所有reads成功比对到基因组上的比例</p>
        <p class="p_img">（6）Genome_Rmdup_Seq_Num：去除了dupication的reads成功比对到基因组上的比例</p>



        <h4>2.3.1 基因组上Reads分布</h4>
        <p>统计Mapped Reads在参考基因组中不同元件（外显子、内含子、基因间区）中的占比，绘制图形如下：</p>
        <div class="single-img">
            <a href=src/img/2.MAP/element.png target="_blank">
                <img src=src/img/2.MAP/element.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 测序Reads的基因组分布图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <p>理论上由于cfRNA组成。。。。。。。。。</p>


        <h4>2.3.2 插入片段检验</h4>
        <p>插入片段可以反应片段长度的离散程度。。。。</p>
        <p>各个样本插入片段长度如下图：</p>
        <div class="single-img">
            <a href="src/img/2.MAP/insertSize.png" target="_blank">
                <img src=src/img/2.MAP/insertSize.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 插入片段检验图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>


        <h4>2.3.3 数据饱和度检验</h4>
        <p>为了评估测序数据是否饱和，即测序数据是否达到了能够测序到基因数目的上限，因为一个物种的基因数目是有限的，且基因转录具有时间和空间特异性，因此随着测序量的增加，检测到的基因数目会趋于饱和。对于高表达量的基因，越容易被检测定量。地表达量基因则需要大量数据才能准确定量。</p>
        <p>使用各样品的Mapped Reads对检测到的不同表达情况的基因数目饱和情况进行模拟，绘制曲线图如下，可查看随着测序数据量的增加，检测到的不同表达量的基因数目是否趋于饱和。</p>
        <div class="single-img">
            <a href=src/img/2.MAP/Saturation.png target="_blank">
                <img src=src/img/2.MAP/Saturation.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 测序数据饱和检验图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>









<!--============================== 基因表达定量 ================================ -->
        <br>
        <h3>2.4 基因表达定量</h3>
        <h4>2.4.1 基因表达定量</h4>
        <p>为了让片段数目能真实地反映转录本表达水平，需要对样品中的Mapped Reads的数目和转录本长度进行归一化。考虑到后续机器学习模型输入的数据要求是符合正太分布的，所以这里使用标准化数据的方式是TMM</p>
        <p>TMM计算公式如下：</p>
        <div text-align="center">
            <p style="text-align:center;">\(TMM = RawCounts * 10^6 / LibrarySize * NormFactor\)</p>
        </div>
        <p class="p_img">公式中，cDNA Fragments表示比对到某一转录本上的片段数目，即双端Reads数目；LibrarySize表示文库的大小，这里表示每个样本的总碱基数目；NormFactor表示标准化因子。</p>
        <p>
            <a href=src/files/2.EXP/TMM_result.txt target="_blank" style="color: red;">
                TMM_result.txt
            </a>
        </p>
        <p class="p_img">注： 第一列表示基因 ID,其余各列表示基因在不同样品中的表达量(TMM)</p>
        

        <h4>2.4.2 不同RNA类型的占比</h4>
        <p>如何判断这些RNA分布在这些不同的RNA上，他们定量的结果是多少，没有用miRNA分析，看下具体定量的基因都map到了哪些类型的基因</p>
        <div class="single-img">
            <a href=src/img/3.EXP/Gene_class.png target="_blank">
                <img src=src/img/3.EXP/Gene_class.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 RNA注释到Gene的类型</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>

        <h4>2.4.2 基因表达水平</h4>
        <p>这些图有什么作用。。。</p>
        <div class="single-img">
            <a href=src/img/3.EXP/box.png target="_blank">    
                <img src=src/img/3.EXP/box.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 基因表达箱线图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <div class="single-img">
            <a href=src/img/3.EXP/density.png target="_blank">    
                <img src=src/img/3.EXP/density.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 基因表达量密度分布曲线</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>

        
        <h4>2.4.3 样本重复性评估</h4>
        <p>基因表达在同一物种的不同个体中也存在着生物学可变性（Biological Variability），所以生物学重复对生物实验尤为重要。尽可能多的生物学重复就可以尽量避免个体差异带来的误差。</p>
        <p>样品间基因表达水平相关性是检验实验可靠性和样本选择是否合理的重要指标。相关系数越接近于 1，表明样品之间表达模式的相似度越高。理想的取样和生物实验要求生物学重复的皮尔森相关系数（Pearson correlation coefficient）的平方（R2）大于 0.92。如若有组内差异较大样本，可能需要考虑剔除异常样本后，再进行后续分析。该项目的相关性图如下：</p>
        <div class="single-img">
            <a href=src/img/3.EXP/density.png target="_blank">    
                <img src=src/img/3.EXP/correlation.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 样本之间相关性热图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <p>PCA（principal components analysis）即主成分分析技术，又称主分量分析，旨在利用降维的思想，把多指标转化为少数几个综合指标。样本数据属于高纬度数据，通过PCA降维，可以将数据降低到二维平面去观察各个样本在主成分中的分布，通常情况下，同组样本之间的间距比较小，聚合在一起，而不同组的样本较为分离。</p>
        <div class="single-img">
            <a href=src/img/3.EXP/PCA.png target="_blank">    
                <img src=src/img/3.EXP/PCA.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 样本降维PCA图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>

        








<!--============================== 差异分析 ================================ -->
        <br>
        <h3>2.5 差异分析</h3>
        <p>基因表达具有时间和空间特异性，在两个不同条件下，表达水平存在显著差异的基因或转录本，称之为差异表达基因（DEG）或差异表达转录本（DET）。在生物学和分子生物学研究中，研究基因表达的差异性通常是为了了解不同条件下基因的调控情况，识别与特定生物学过程或疾病相关的差异表达基因，以及揭示潜在的生物学机制。</p>
        <h4>2.5.1 差异分析结果统计</h4>
        <p>样本组之间通过DEseq2进行差异分析，获取两个生物学条件之间的差异表达基因集；对于没有生物学重复的样本使用edgeR[25]进行差异分析。在差异表达基因检测过程中，将Fold Change≥2且FDR<0.01作为筛选标准。差异倍数（Fold Change）表示两样品（组）间表达量的比值。错误发现率（False Discovery Rate，FDR）是通过对差异显著性p值（p-value）进行校正得到的 。</p>
        <p>
            <a href=src/files/3.DEG/Treatment_vs_Control_DEseq2.xls target="_blank" style="color: red;">
                Treatment_vs_Control_DEseq2.xls
            </a>
        </p>
        <p>差异表达基因统计表格如下：</p>
        <div class="scrollable-div">
            <table>
                <tr>
                    <th>DEG Group</th>
                    <th>DEG gene number</th>
                    <th>up number</th>
                    <th>down number</th>
                </tr>
                
                <tr>
                    <td>A_vs_B</td>
                    <td>2000</td>
                    <td>1000</td>
                    <td>1000</td>
                </tr>
                
            </table>
        </div>
        <p class="p_img">注：DEG Group：差异表达基因集名称；DEG gene number：差异表达基因数目；up number：上调基因的数目；down number：下调基因数目。</p>
        <p>通过火山图（Volcano Plot）可以快速查看基因在两个（组）样本中水平的差异，观察上下调基因的个数，和差异大小分布。横轴表示基因的表达差异（通常是对数变换的折叠变化），纵轴表示基因的显著性（通常是统计学上的显著性，例如负对数的p值）</p>         
        <div class="single-img">
            <a href=src/img/4.DEG/Treatment_vs_Control_volcano.png target="_blank">    
                <img src=src/img/4.DEG/Treatment_vs_Control_volcano.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异火山图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>


        <h4>2.5.2 差异聚类分析</h4>
        <p>对筛选出来的差异基因做层次聚类分析</p>
        <div class="single-img">
            <a href=src/img/4.DEG/DEG_heatmap.png target="_blank">    
                <img src=src/img/4.DEG/DEG_heatmap.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异聚类热图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>


        <h4>2.5.3 差异分析功能注释和富集分析</h4>
        <p>描述我们是如何做的GO、KEGG注释。。。。富集分析有什么用。。。</p>
        <p>对差异基因进行GO、KEGG数据库的功能注释，各组差异基因注释结果见下表：</p>
        <div class="scrollable-div">
            <table>
                <tr>
                    <th>DEG Group</th>
                    <th>Total</th>
                    <th>GO</th>
                    <th>KEGG</th>
                </tr>
                
                <tr>
                    <td>A_vs_B</td>
                    <td>2000</td>
                    <td>1000</td>
                    <td>52</td>
                </tr>
                
            </table>
        </div>
        <p class="p_img">注：。。。。</p>
        <h5>GO富集分析</h5>
        <p>GO数据库介绍。。。,topGO结构介绍，注释库的大功能介绍</p>
        <div class="single-img">
            <a href=src/img/4.DEG/GO_classification.png target="_blank">    
                <img src=src/img/4.DEG/GO_classification.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因GO注释图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <div class="single-img">
            <a href=src/img/4.DEG/GO_enrichment.png target="_blank">    
                <img src=src/img/4.DEG/GO_enrichment.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因GO富集气泡图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <h5>KEGG富集分析</h5>
        <p>KEGG数据库介绍，通路等等。。，我们使用的是人专门的KEGG通路</p>
        <div class="single-img">
            <a href=src/img/4.DEG/KEGG_classification.png target="_blank">    
                <img src=src/img/4.DEG/KEGG_classification.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因KEGG注释图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <div class="single-img">
            <a href=src/img/4.DEG/KEGG_enrichment.png target="_blank">    
                <img src=src/img/4.DEG/KEGG_enrichment.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因KEGG富集气泡图</p>
        <p class="p_img">注：横坐标是。。。纵坐标是。。。</p>
        <div class="single-img">
            <a href=src/img/4.DEG/kegg_map.png target="_blank">    
                <img src=src/img/4.DEG/kegg_map.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因KEGG通路图</p>
        <p class="p_img">注：这个图什么含义</p>


        <h4>2.5.4 差异基因蛋白互作网络</h4>
        <p>利用 string 数据库对上下调差异基因分析蛋白互作关系，差异基因较多的情况下，上下调各选取 500 个差异基因进行互作关系的分析，蛋白互作表格展示如下。。。。</p>
        <div class="scrollable-div">
            <table>
                <tr>
                    <th>preferredName_A</th>
                    <th>preferredName_B</th>
                    <th>stringId_A</th>
                    <th>stringId_B</th>
                    <th>score</th>
                </tr>
                
                <tr>
                    <td>PDK4</td>
                    <td>ACAA2</td>
                    <td>9606.ENSP00000005178</td>
                    <td>9606.ENSP00000285093</td>
                    <td>0.468</td>
                </tr>
                
                <tr>
                    <td>PDK4</td>
                    <td>ECHS1</td>
                    <td>9606.ENSP00000005178</td>
                    <td>9606.ENSP00000357535</td>
                    <td>0.495</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>MT2A</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000245185</td>
                    <td>0.432</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>MYC</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000478887</td>
                    <td>0.438</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>SRC</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000362680</td>
                    <td>0.456</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>ARG1</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000349446</td>
                    <td>0.476</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>MYD88</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000498321</td>
                    <td>0.553</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>MT1E</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000328137</td>
                    <td>0.568</td>
                </tr>
                
                <tr>
                    <td>HMOX1</td>
                    <td>SLC7A11</td>
                    <td>9606.ENSP00000216117</td>
                    <td>9606.ENSP00000280612</td>
                    <td>0.61</td>
                </tr>
                
                <tr>
                    <td>PYGB</td>
                    <td>ACAA2</td>
                    <td>9606.ENSP00000216962</td>
                    <td>9606.ENSP00000285093</td>
                    <td>0.403</td>
                </tr>
                
                <tr>
                    <td>GLA</td>
                    <td>TBX19</td>
                    <td>9606.ENSP00000218516</td>
                    <td>9606.ENSP00000356795</td>
                    <td>0.417</td>
                </tr>
                
                <tr>
                    <td>GLA</td>
                    <td>SORT1</td>
                    <td>9606.ENSP00000218516</td>
                    <td>9606.ENSP00000256637</td>
                    <td>0.683</td>
                </tr>
                
                <tr>
                    <td>ID2</td>
                    <td>MYC</td>
                    <td>9606.ENSP00000234091</td>
                    <td>9606.ENSP00000478887</td>
                    <td>0.425</td>
                </tr>
                
                <tr>
                    <td>ID2</td>
                    <td>ID1</td>
                    <td>9606.ENSP00000234091</td>
                    <td>9606.ENSP00000365280</td>
                    <td>0.516</td>
                </tr>
                
                <tr>
                    <td>MT2A</td>
                    <td>MYC</td>
                    <td>9606.ENSP00000245185</td>
                    <td>9606.ENSP00000478887</td>
                    <td>0.515</td>
                </tr>
                
                <tr>
                    <td>MT2A</td>
                    <td>NREP</td>
                    <td>9606.ENSP00000245185</td>
                    <td>9606.ENSP00000378996</td>
                    <td>0.557</td>
                </tr>
                
                <tr>
                    <td>MT2A</td>
                    <td>MT1E</td>
                    <td>9606.ENSP00000245185</td>
                    <td>9606.ENSP00000328137</td>
                    <td>0.829</td>
                </tr>
                
                <tr>
                    <td>HBA2</td>
                    <td>HBB</td>
                    <td>9606.ENSP00000251595</td>
                    <td>9606.ENSP00000494175</td>
                    <td>0.999</td>
                </tr>
                
                <tr>
                    <td>SORT1</td>
                    <td>LIFR</td>
                    <td>9606.ENSP00000256637</td>
                    <td>9606.ENSP00000398368</td>
                    <td>0.475</td>
                </tr>
                
                <tr>
                    <td>SORT1</td>
                    <td>DNAJC6</td>
                    <td>9606.ENSP00000256637</td>
                    <td>9606.ENSP00000360108</td>
                    <td>0.561</td>
                </tr>
                
                <tr>
                    <td>DTX1</td>
                    <td>MYC</td>
                    <td>9606.ENSP00000257600</td>
                    <td>9606.ENSP00000478887</td>
                    <td>0.408</td>
                </tr>
                
                <tr>
                    <td>POLQ</td>
                    <td>DNA2</td>
                    <td>9606.ENSP00000264233</td>
                    <td>9606.ENSP00000351185</td>
                    <td>0.682</td>
                </tr>
                
                <tr>
                    <td>F13A1</td>
                    <td>FCGR3A</td>
                    <td>9606.ENSP00000264870</td>
                    <td>9606.ENSP00000356946</td>
                    <td>0.448</td>
                </tr>
                
                <tr>
                    <td>F13A1</td>
                    <td>C1QC</td>
                    <td>9606.ENSP00000264870</td>
                    <td>9606.ENSP00000363770</td>
                    <td>0.466</td>
                </tr>
                
                <tr>
                    <td>F13A1</td>
                    <td>A2M</td>
                    <td>9606.ENSP00000264870</td>
                    <td>9606.ENSP00000323929</td>
                    <td>0.572</td>
                </tr>
                
                <tr>
                    <td>RHOB</td>
                    <td>MYC</td>
                    <td>9606.ENSP00000272233</td>
                    <td>9606.ENSP00000478887</td>
                    <td>0.403</td>
                </tr>
                
                <tr>
                    <td>RHOB</td>
                    <td>ZFP36L1</td>
                    <td>9606.ENSP00000272233</td>
                    <td>9606.ENSP00000337386</td>
                    <td>0.412</td>
                </tr>
                
                <tr>
                    <td>RHOB</td>
                    <td>DUSP6</td>
                    <td>9606.ENSP00000272233</td>
                    <td>9606.ENSP00000279488</td>
                    <td>0.416</td>
                </tr>
                
                <tr>
                    <td>RHOB</td>
                    <td>MAP1A</td>
                    <td>9606.ENSP00000272233</td>
                    <td>9606.ENSP00000300231</td>
                    <td>0.49</td>
                </tr>
                
                <tr>
                    <td>RHOB</td>
                    <td>SRC</td>
                    <td>9606.ENSP00000272233</td>
                    <td>9606.ENSP00000362680</td>
                    <td>0.539</td>
                </tr>
                
                <tr>
                    <td>DUSP6</td>
                    <td>ID1</td>
                    <td>9606.ENSP00000279488</td>
                    <td>9606.ENSP00000365280</td>
                    <td>0.483</td>
                </tr>
                
                <tr>
                    <td>ACAA2</td>
                    <td>ASS1</td>
                    <td>9606.ENSP00000285093</td>
                    <td>9606.ENSP00000361471</td>
                    <td>0.423</td>
                </tr>
                
                <tr>
                    <td>ACAA2</td>
                    <td>SUCLG2</td>
                    <td>9606.ENSP00000285093</td>
                    <td>9606.ENSP00000419325</td>
                    <td>0.88</td>
                </tr>
                
                <tr>
                    <td>ACAA2</td>
                    <td>ECHS1</td>
                    <td>9606.ENSP00000285093</td>
                    <td>9606.ENSP00000357535</td>
                    <td>0.975</td>
                </tr>
                
                <tr>
                    <td>CD109</td>
                    <td>ITGA2</td>
                    <td>9606.ENSP00000287097</td>
                    <td>9606.ENSP00000296585</td>
                    <td>0.419</td>
                </tr>
                
                <tr>
                    <td>ITGA2</td>
                    <td>FCGR3A</td>
                    <td>9606.ENSP00000296585</td>
                    <td>9606.ENSP00000356946</td>
                    <td>0.535</td>
                </tr>
                
                <tr>
                    <td>ITGA2</td>
                    <td>SRC</td>
                    <td>9606.ENSP00000296585</td>
                    <td>9606.ENSP00000362680</td>
                    <td>0.812</td>
                </tr>
                
                <tr>
                    <td>MS4A7</td>
                    <td>C1orf162</td>
                    <td>9606.ENSP00000300184</td>
                    <td>9606.ENSP00000344218</td>
                    <td>0.45</td>
                </tr>
                
                <tr>
                    <td>MS4A7</td>
                    <td>C1QC</td>
                    <td>9606.ENSP00000300184</td>
                    <td>9606.ENSP00000363770</td>
                    <td>0.457</td>
                </tr>
                
                <tr>
                    <td>EZH2</td>
                    <td>SRC</td>
                    <td>9606.ENSP00000320147</td>
                    <td>9606.ENSP00000362680</td>
                    <td>0.469</td>
                </tr>
                
            </table>
        </div>
        <p class="p_img">注：表格意思。。。。</p>
        <p>Cytoscape作图可视化。。。。</p>
        <div class="single-img">
            <a href=src/img/4.DEG/PPI.png target="_blank">    
                <img src=src/img/4.DEG/PPI.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 差异基因蛋白互作网络图</p>
        <p class="p_img">注：这个图什么含义</p>


        <h4>2.5.5 GSEA富集分析</h4>
        <p>介绍什么是GSEA分析,我们不仅会使用GO和KEGG比较显著的通路，同时还会使用MsigDB中疾病数据库中已经收录的数据库去做GSEA</p>
        <div class="single-img">
            <a href=src/img/4.DEG/gseaplot.png target="_blank">    
                <img src=src/img/4.DEG/gseaplot.png alt="">
            </a>
        </div>
        <p style="text-align: center;">图 显著的通路或功能的GSEA结果图</p>
        <p class="p_img">注：横坐标是什么。。。纵坐标是什么。。。上下图都是什么含义。。。</p>










        
<!--============================== lncRNA分析 ================================ -->
        <h3>2.6 lncRNA共表达分析</h3>
        <p>starbase已知数据库介绍。。。。lncRNA与mRNA的互作表格和互作网络展示</p>
        <p>下面是差异的lncRNA与mRNA的关系表</p>
        <div class="single-img">
            <a href=src/img/5.lncRNA/lncRNA-mRNA.png target="_blank">    
                <img src=src/img/5.lncRNA/lncRNA-mRNA.png alt="">
            </a>
        </div>
        <p class="p_img">注：表格内容注释</p>










<!--============================== circRNA分析 ================================ -->
        <h3>2.7 circRNA鉴定</h3>
        <p>circRNA介绍</p>
        <h4>2.7.1 CIRCexplorer检测</h4>
        <p>CIRCexplorer2鉴定并注释新的circRNA，其通过。。。。。</p>
        <div class="single-img">
            <a href=src/img/6.circRNA/circRNA-identify.png target="_blank">    
                <img src=src/img/6.circRNA/circRNA-identify.png alt="">
            </a>
        </div>
        <p class="p_img">注：表格内容注释</p>
        

        <h4>2.7.2 circRNA外显子分布</h4>
        <div class="single-img">
            <a href=src/img/6.circRNA/circRNA_exon_num.png target="_blank">    
                <img src=src/img/6.circRNA/circRNA_exon_num.png alt="">
            </a>
        </div>
        <p class="p_img">注：横坐标是什么，纵坐标是什么</p>


        <h4>2.7.3 circRNA注释</h4>
        <h5>circbase数据库注释</h5>
        <p>cicrbase数据库介绍。。。</p>
        <div class="single-img">
            <a href=src/img/6.circRNA/circRNA_Anno.png target="_blank">    
                <img src=src/img/6.circRNA/circRNA_Anno.png alt="">
            </a>
        </div>
        <p class="p_img">注：横坐标是什么，纵坐标是什么</p>
    









<!--============================== 总结 ================================ -->
        <hr class="solid-line" align="center">
        <br>
        <h2>参考文献</h2>
        <p>....</p>

    </div>
</body>
</html>